def raise_exception(exception):
    raise exception